conda/post-link.sh

```
@@ -1,38 +1,184 @@
+import sys
+try:
+    import torch
+    available = torch.cuda.is_available()
+except Exception:
+    available = None
+
+sys.stdout.write(
+    "True" if available else ("False" if available is not None else "Unavailable")
+)
+PY
+    2>>"${OUT_STREAM}")
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)